src/main/java/org/aion/wallet/util/SyncStatusFormatter.java

```diff
@@ -0,0 +1,42 @@
+package org.aion.wallet.util;
+
+import org.aion.api.server.types.SyncInfo;
+
+public class SyncStatusFormatter {
+
+    public static final int SECONDS_IN_A_MINUTE = 60;
+    public static final int SECONDS_IN_A_HOUR = 3600;
+    public static final int SECONDS_IN_A_DAY = 86400;
+    public static final String UNDEFINED = "Undefined";
+    public static final String UP_TO_DATE = "Up to date";
+
+    public static String formatSyncStatus(SyncInfo syncInfo) {
+        if(syncInfo != null) {
+            if(syncInfo.networkBestBlkNumber > 0) {
+                long seconds = (syncInfo.networkBestBlkNumber - syncInfo.chainBestBlkNumber) * 10;
```

Comment:
10, 24, 60 can also be constants

Reply:
There's a `WalletUtils.BLOCK_MINING_TIME` constant so you could rename that to `WalletUtils.BLOCK_MINING_TIME_MILLIS` and add `WalletUtils.BLOCK_MINING_TIME_SECONDS` or something like that